components/git/land.js

```diff
@@ -42,6 +45,21 @@ const landOptions = {
     describe: 'Land a backport PR onto a staging branch',
     default: false,
     type: 'boolean'
+  },
+  skipChecks: {
```

Comment:
We should add this option to documentation as well, perhaps with an accompanying example :)